app/src/androidTest/java/org/simple/clinic/storage/migrations/Migration57AndroidTest.kt

```diff
@@ -0,0 +1,343 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.sqlite.db.SupportSQLiteDatabase
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.simple.clinic.TestClinicApp
+import org.simple.clinic.TestData
+import org.simple.clinic.assertValues
+import org.simple.clinic.insert
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.patient.Patient
+import org.simple.clinic.patient.PatientProfile
+import org.simple.clinic.patient.PatientStatus
+import org.simple.clinic.patient.ReminderConsent
+import org.simple.clinic.patient.SyncStatus
+import org.simple.clinic.patient.businessid.BusinessId
+import org.simple.clinic.patient.businessid.Identifier
+import org.simple.clinic.string
+import java.util.UUID
+import javax.inject.Inject
+
+class Migration57AndroidTest : BaseDatabaseMigrationTest(56, 57) {
+
+  @Inject
+  lateinit var testData: TestData
+
+  @Before
+  override fun setUp() {
+    super.setUp()
+    TestClinicApp.appComponent().inject(this)
+  }
+
+  @Suppress("IllegalIdentifier")
+  @Test
+  fun migrating_to_57_should_mark_all_patients_with_blank_identifiers_as_sync_pending() {
```

Comment:
This just needs one more patient with ID that has non-blank value.

Reply:
Done.